### PR TITLE
Content changes from Kate

### PR DESCRIPTION
**Pull request by khandelwal**
@kategarklavs did another pass on our content, I merely am implementing those changes here. So, new content!

### CODE REVIEW

File: foia_hub/templates/learn.html [hunk 1]
```diff
@@ -68,16 +72,21 @@ <h2 id="ask" tabindex="-1" class="question">What can I ask for?</h2>
...
       <div class="answer">
-        <p>There is no specific form that must be used to make a request. All
-        you need is to do is write a description of the information you are
-        seeking and comply with any agency-specific requirements. Those
-        requirements are set out in agencies’ FOIA regulations.</p>
+      
+       <p>You do not need to use a specific form to make a request. All you
+       need is to do is write a description of the information you are seeking
+       and comply with any agency-specific requirements, which are described in
+       agencies' FOIA regulations. ONce you've written your description and
```
Comment (ramirezg) [inline foia_hub/templates/learn.html]: s/ONce/Once

File: foia_hub/templates/learn.html [hunk 2]
```diff
@@ -143,26 +163,31 @@ <h2 id="when" tabindex="-1" class="question">How long does it take to fulfill my
...
+        response within 20 business days, but response times can vary based on
+        an agency’s workload and the complexity of your appeal. </p>
+
+        <p> Sometimes an appeal isn’t the best route. Before filing an appeal,
+        you might want to try contacting the agency that handled the request.
+        You can reach out directly to your FOIA contact, or you can contact the
+        agency’s FOIA public liaison, who will offer guidance throughout the
+        FOIA process and help resolve any disputes. Every agency provides
+        information about their FOIA public liaison on their <a href="{{
+        url('agencies') }}">own page or component pagesA</a>. </p>
```
Comment (ramirezg) [inline foia_hub/templates/learn.html]: typo, extra A

**Comment by khandelwal** [comment]
The first batch of typos fixed!

**Comment by ramirezg** [comment]
Looks great. I'll merge after Travis finishes.

**Pull request merged**